### PR TITLE
Epic Subtle Highlight design test

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?

Both the highlight and the subtle variant beat the control so we want to combine them into one and see if that combines the benefit of both of them. 

## Screenshots
screenshot at mar 10 07-26-51

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js
Comment (katebee) [inline static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js]: nitpick: where did the spaaaaace go?

`variants: ['control', 'subtle_highlight']`

File: common/app/conf/switches/ABTestSwitches.scala
Comment (katebee) [inline common/app/conf/switches/ABTestSwitches.scala]: JS test expiries this Friday, is this by design (since test expiries don't have to match their Switch), or does it need to be extended (like the Switch)?

**Review by katebee — APPROVED** [review]
May want to check JS expiry, otherwise LGTM 👍 🚢

**Comment by Mullefa** [comment]
@guardian/contributions 

Do not merge yet as there's going to be 5 new variants to run against the control (Mario will be providing the designs). Also, we need to confirm the results of the first iteration are statistically significant (although I'm sure they are).

**Comment by dominickendrick** [comment]
This has been superseded by https://github.com/guardian/frontend/pull/16132

**Pull request closed without merging**
